pybedtools/bedtool.py

```diff
@@ -2537,17 +2575,22 @@ def jaccard(self):
         other="b",
         does_not_return_bedtool=_reldist_output_handler,
     )
-    def reldist(self):
+    def reldist(self, *args, **kwargs) -> BedTool | dict[str, Any]: # type: ignore
         """
         If detail=False, then return a dictionary with keys (reldist, count,
         total, fraction), which is the summary of the bedtools reldist.
 
         Otherwise return a BedTool, with the relative distance for each
         interval in A in the last column.
         """
+        if "detail" in kwargs:
```

Comment:
reldist can output a Bedtool object or a dict if detail kwarg is False. I added a warning to the user of when this is the case